.editorconfig

```diff
@@ -0,0 +1,10 @@
+# For more information about the properties used in this file,
+# please see the EditorConfig documentation:
+# http://editorconfig.org
+
+[*]
```

Comment:
So if prettier inherits these rules like you say, we should make sure these matchup with what's in prettier defaults. I know indent_size does, not sure about the others.

Reply:
Add to that, IMHO: I'd also be explicit in the `.prettierrc.json` since I prefer more clarity than obscurity in general (hunting down defaults and etc). At least for the ones we care about. So that means 

1. Codifying the defaults into the config for the values we care about (granted, verbose as it is)
2. Applying those values here as well

That said, you commented on the file selector/glob. Maybe worth just limiting to js, json, html and css? Excluding MD and etc.

Reply:
Looks like prettier will ignore these (which is what I want) because the default prettier setting is `options.editorconfig=false`:

https://prettier.io/docs/en/configuration.html#editorconfig

I'd prefer not to be explicit in the prettierconfig as I am happy with the out of the box defaults. I need to remove the printWidth override we have.

Reply:
Per our offline discussion, we'll keep that print width in place. Instead, I've documented that we're just intentionally aligning `.editorconfig` with our prettier configuration (which is barebones) to help increase surface area of IDEs that'll respect repo code style guidelines.

I've also explicitly omitted `options.editorconfig` from `.prettierrc.json` in order to maintain this minimalism. Again, the goal is just to align `.editorconfig` with prettier and not the other way around. 🙂 I'll reiterate this in my comment in the PR as well.